container/go/fast_puller.go

```diff
@@ -0,0 +1,47 @@
+package main
```

Comment:
I guess you probably want to delete this file as container/go/cmd/puller/puller.go is the new one?